1. Metataxonomics
=================

*Metataxonomics* allows us to investigate the composition of a microbial community using amplicons from a targeted marker gene in order to make taxonomic inferences. A variety of tools and analysis pipelines have been developed to deal with this type of data. Some of the most popular ones are QIIME and Mothur. *QIIME* provides a set of integratable scripts for analyzing raw microbial DNA samples including taxonomic classification using marker genes such as 16S rRNA, but allowing flexible pipelines to be constructed. Although *Mothur* was initially designed to target the microbial ecology community, it has since been adopted by the human microbiome community as well. It provides an extensible package with functionality accessible through a domain-specific language. Like QIIME, Mothur is also a metataxonomic tool, focusing on marker genes such as 16S rRNA.

1.1 16S
-------

**General tutorials**

- `Using QIIME to analyze 16S rRNA gene sequences from Microbial Communities <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3249058/>`_

**Illumina**

- `Official Mothur MiSeq SOP <http://www.mothur.org/wiki/MiSeq_SOP>`_
- `Mealybugs's metagenomic pipeline using Mothur <http://mealybugs-metagenomics.readthedocs.org/en/latest/>`_
- `Illumina Overview Tutorial using QIIME: Moving Pictures of the Human Microbiome <http://nbviewer.jupyter.org/github/biocore/qiime/blob/1.9.1/examples/ipynb/illumina_overview_tutorial.ipynb>`_

**454**

- `Official Mothur 454 SOP <http://www.mothur.org/wiki/454_SOP>`_
- `454 Overview Tutorial using QIIME: de novo OTU picking and diversity analyses using 454 data <http://qiime.org/tutorials/tutorial.html>`_

1.2 18S
-------

- `Analysis of 18S data using QIIME <http://qiime.org/tutorials/processing_18S_data.html>`_

1.3 ITS
-------

- `Fungal ITS QIIME analysis tutorial <http://nbviewer.jupyter.org/github/biocore/qiime/blob/1.9.1/examples/ipynb/Fungal-ITS-analysis.ipynb>`_

